ci/prow/config_knative.yaml

```diff
@@ -172,6 +172,10 @@ presubmits:
       dot-dev: true
     - go-coverage: true
       dot-dev: true
+    - custom-test: integration-tests-on-latest-serving
+      always_run: false
+      optional: true
+      full-command: "./test/e2e-tests-latest-serving.sh"
```

Comment:
Don't use full-command, you're skipping all the presubmit checks.

Reply:
Done, changed to command

Reply:
It should be args, including a `--run-tests`, just like serving does.

Reply:
Changed presubmit job to that format. Also tried to change the periodic job, but seems it's not supported.